test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/DefaultTemplatesUtilities.cs

```diff
@@ -42,6 +42,10 @@ public ObjectTemplateModel()
             public object ComplexInnerModel { get; set; }
         }
 
+        public class DerivedObjectTemplateModel : ObjectTemplateModel
```

Comment:
Why is this here instead of `HtmlHelperTest`?

Reply:
Because other tests in HtmlHelperTests were using these models.

Reply:
Please move this particular one to `HtmlHelperTest`. The ones here are used in multiple test classes.